sfja/src/components/AddStudent/Input.js

```diff
@@ -158,7 +159,7 @@ class Input extends React.Component {
               label="Date picker inline"
               value={dob}
               onChange={(ev) => {
-                this.setState({dob: ev});
+                this.setState({dob: ev.toLocaleDateString()});
```

Comment:
There will be an issue here! Can anyone spot it ;)

Reply:
is ev a valid date object?

Reply:
is ev a valid date object?

Reply:
It usually is. Did you test the branch? It should have no issues in the expected case.